dataland-frontend/src/components/forms/parts/elements/basic/ExtendedDataPointFormField.vue

```diff
@@ -99,15 +99,15 @@
               description="The level of confidence associated to the value."
               :is-required="isDataQualityRequired"
             />
-            <FormKit
-              type="select"
-              v-model="qualityValue"
+            <SingleSelectFormElement
               name="quality"
+              v-model="qualityValue"
               :disabled="!isDataQualityRequired"
               :validation="isDataQualityRequired ? 'required' : ''"
               validation-label="Data quality"
               placeholder="Data quality"
               :options="computeQualityOption"
+              isRequired
```

Comment:
manchmal isRequired, manchmal :required="true", manchmal is-required="true", funktioniert das alles gleich?
dann würde ich das einheitlich machen.

Reply:
bei SingleSelectFormField ist es required, bei SingleSelectFormElement ist es is-required. 
Irgendwie hatte es Probleme gegeben wenn es im SingleSelectFormElement required heißt
vielleicht dann überall is-required nennen?